vertigo-bot-designer/src/main/javagen/io/vertigo/chatbot/designer/builder/chatbot/ChatbotPAO.java

```diff
@@ -0,0 +1,62 @@
+package io.vertigo.chatbot.designer.builder.chatbot;
+
+import javax.inject.Inject;
+
+import io.vertigo.core.node.Node;
+import io.vertigo.core.lang.Assertion;
+import io.vertigo.core.lang.Generated;
+import io.vertigo.datamodel.task.TaskManager;
+import io.vertigo.datamodel.task.definitions.TaskDefinition;
+import io.vertigo.datamodel.task.model.Task;
+import io.vertigo.datamodel.task.model.TaskBuilder;
+import io.vertigo.datastore.impl.dao.StoreServices;
+
+/**
+ * This class is automatically generated.
+ * DO NOT EDIT THIS FILE DIRECTLY.
+ */
+ @Generated
+public final class ChatbotPAO implements StoreServices {
+	private final TaskManager taskManager;
+
+	/**
+	 * Constructeur.
+	 * @param taskManager Manager des Task
+	 */
+	@Inject
+	public ChatbotPAO(final TaskManager taskManager) {
+		Assertion.check().isNotNull(taskManager);
+		//-----
+		this.taskManager = taskManager;
+	}
+
+	/**
+	 * Creates a taskBuilder.
+	 * @param name  the name of the task
+	 * @return the builder 
+	 */
+	private static TaskBuilder createTaskBuilder(final String name) {
+		final TaskDefinition taskDefinition = Node.getNode().getDefinitionSpace().resolve(name, TaskDefinition.class);
+		return Task.builder(taskDefinition);
+	}
+
+	/**
+	 * Execute la tache TkRemoveAllChaPerRightByBotId.
+	 * @param botId Long
+	*/
+	@io.vertigo.datamodel.task.proxy.TaskAnnotation(
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.